src/activities/jumbled_words/Jumbled_words.qml

```diff
@@ -0,0 +1,486 @@
+                font {
+                    pointSize: (44 + ApplicationSettings.baseFontSize) * ApplicationInfo.fontRatio
+                    capitalization: ApplicationSettings.fontCapitalization
+                    weight: Font.Bold
+                    family: GCSingletonFontLoader.fontLoader.name
+                    letterSpacing: ApplicationSettings.fontLetterSpacing
+                    wordSpacing: 10
+                }
+
+                cursorDelegate: Rectangle {
+                    id: cursor
+                    width: 10
+                    height: parent.cursorRectangle.height
+                    color: 'white'
+                    SequentialAnimation on opacity {
+                        running: true
+                        loops: Animation.Infinite
+                        PropertyAnimation {
+                            to: 0.2
+                            duration: 1000
+                        }
+                        PropertyAnimation {
+                            to: 1
+                            duration: 1000
+                        }
+                    }
+
+                }
+                function insertText(text) {
+                    edit.insert(cursorPosition, text)
+                }
+                function backspace() {
+                    if(cursorPosition > 0) {
+                        moveCursorSelection(cursorPosition - 1, TextEdit.SelectCharacters)
+                        cut()
+                    }
+                }
+                function formatLineWith(tag) {
+                    var text = getText(0, length)
+                    var initialPosition = cursorPosition
+                    var first = cursorPosition - 1
+                    for(; first >= 0; first--) {
+                        if(text.charCodeAt(first) === 8233)
+                            break
+                    }
+                    first++
+                    var last = cursorPosition
+                    for(; last < text.length; last++) {
+                        if(text.charCodeAt(last) === 8233)
+                            break
+                    }
+                    var line = getText(first, last)
+                    remove(first, last)
+                    insert(first, '<' + tag + '>' + line + '</' + tag + '>')
+                    cursorPosition = initialPosition
+                }
+            }
+        }
+
+        Rectangle {
+            id: submit
+            anchors.fill: submitText
+            border.color: "#FFFFFFFF"
+            border.width: 2
+            height:submitText.height
+            color: "#000065"
+            opacity: 0.31
+            radius: 10
+
+            MouseArea {
+                id: mouseAreaSubmit
+                anchors.fill: parent
+                hoverEnabled: ApplicationInfo.isMobile ? false : true
+                onClicked: {
+                    if(Activity.checkAnswer())
+                    {
+                        edit.text=""
```

Comment:
you should not set the values here but probably use a boolean hintDisplayed and the hintText.text should be something like: hintDisplayed ? theHint : qsTr("Hint")